stonesoup/types/tests/test_angle.py

```diff
@@ -108,3 +109,92 @@ def test_misc(self, Class, func):
         assert b1 % 0.1 == b2 % 0.1
         assert +b1 == +b2
         assert approx(-b1) == -b2
+
+
+scenario3 = ('longitude', {'Class': Longitude, 'func': mod_longitude})
+scenario4 = ('latitude', {'Class': Latitude, 'func': mod_latitude})
+
+
+class TestAngleDegrees:
```

Comment:
I would prefer `class_` here to `Class` here, as I think it may be less confusing.

Reply:
@etfrogers-dstl Thanks for the review. Following discussions during the telecons, the PR will be modified so that it deals with radians, instead of degrees, so I would be grateful if you could have another look once this is done.

But, in relation to your comment, I have not preference on what keyword we use. This code was originally written by @DaveKirkland , so I have just copied and pasted his tests here. I don't mind correcting this if you/others find it appropriate.